### DISCUSSION TITLE
[FEATURE] Replace LGTM with CodeQL

### DISCUSSION THREAD

**Discussion by Panquesito7** [TheAlgorithms/C-Plus-Plus#1462] [Ideas] [answered]
[LGTM](https://lgtm.com) is a nice scanning tool. [CodeQL](https://github.com/github/codeql) is what powers [LGTM](https://lgtm.com). However, [LGTM](https://lgtm.com) takes too long to check for new/fixed alerts. [CodeQL](https://github.com/github/codeql) **works much more [faster](https://github.com/TheAlgorithms/website/pull/24)**, and [LGTM](https://lgtm.com) seems to only use [CodeQL](https://github.com/github/codeql). You can also see [CodeQL](https://github.com/github/codeql) runs on commits, and [LGTM](https://lgtm.com) doesn't.

Please tell us what do you think about this.
We will wait for your response. 🙂

Thank you.

**Comment by cclauss** [comment]
Faster turnarounds are better.

**Comment by Panquesito7** [comment]
@ayaankhan98, what do you think? 🙂

**Reply by ayaankhan98** [reply]
codeQL is better.

**Comment by Panquesito7** [comment, marked as answer]
https://github.com/TheAlgorithms/C-Plus-Plus/pull/1465
